assets/docs/neso613/ASR_TFLite/1.md

```diff
@@ -0,0 +1,10 @@
+# Placeholder neso613/ASR_TFLite/1
```

Comment:
Since your TFLite document references the `neso613/ASR_TFLite/pre_trained_models/1.md` placeholder but not this once, would you mind removing this file? We don't need it to display your model on tfhub.dev :)

Reply:
I don't mind but I think, with a folder there must be 1.md. Isn't it?

Reply:
No it's safe to just delete this file. This model also does not contain a file in every parent directory: https://github.com/tensorflow/tfhub.dev/tree/master/assets/docs/inaturalist/models/vision/embedder/inaturalist_V2